src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars > .stars__star:nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 > .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-last-child(n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-last-child(n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-last-child(n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-last-child(n+1) {
+  background-image: url(images/star-active.svg);
+}
```

let's combine these rules like this

```suggestion
.stars--4 > .stars__star:nth-last-child(n+2),
.stars--5 > .stars__star:nth-last-child(n+1),
.etc {
  background-image: url(images/star-active.svg);
}
```